compiler/rustc_codegen_gcc/src/intrinsic/mod.rs

```diff
@@ -74,8 +74,12 @@ fn get_simple_intrinsic<'gcc, 'tcx>(
         sym::fabsf64 => "fabs",
         sym::minnumf32 => "fminf",
         sym::minnumf64 => "fmin",
+        sym::minimumf32 => "fminimum",
+        sym::minimumf64 => "fminimumf",
```

Comment:
I believe you need to swap those as, according to the document you posted, it is `fminimumf` that takes a `float` (what we consider a `f32` for now in some cases).

Same for below.

Reply:
Indeed my bad, fixed; and also added the `f128` version with the `l` suffix.